Enigmatry.Entry.TemplatingEngine.Fluid/FluidTemplatingEngine.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Enigmatry.Entry.Core.Helpers;
+using Enigmatry.Entry.TemplatingEngine.Liquid.CustomFilters;
+using Fluid;
+using Fluid.Values;
+using Microsoft.Extensions.Logging;
+using System.Globalization;
+
+namespace Enigmatry.Entry.TemplatingEngine.Liquid;
+
+public class FluidTemplatingEngine : ITemplatingEngine
+{
+    private readonly ILogger<FluidTemplatingEngine> _logger;
+    private readonly IFluidTemplateProvider _templateProvider;
+    private readonly IEnumerable<ICustomFluidFilter> _fluidFilters;
+    private readonly FluidTemplateEngineOptions _options;
+
+
+    public FluidTemplatingEngine(ILogger<FluidTemplatingEngine> logger, IFluidTemplateProvider templateProvider,
+        IEnumerable<ICustomFluidFilter> fluidFilters, FluidTemplateEngineOptions options)
+    {
+        _logger = logger;
+        _templateProvider = templateProvider;
+        _fluidFilters = fluidFilters;
+        _options = options;
+    }
+
+    public async Task<string> RenderAsync<T>(string path, T model)
+    {
+        ArgumentNullException.ThrowIfNull(path);
+        ArgumentNullException.ThrowIfNull(model);
+
+        var fluidTemplate = _templateProvider.GetTemplate(path);
+        var options = new TemplateOptions
+        {
+            MemberAccessStrategy =
+                new LoggingUnsafeMemberAccessStrategy(_logger) { MemberNameStrategy = _options.MemberNameStrategy },
+            CultureInfo = _options.CultureInfo
+        };
+
+        if (_options.ConvertEnumToString)
+        {
+            options.ValueConverters.Add(value => value is Enum e ? new StringValue(e.GetDescription()) : null);
+        }
+
+        options.ValueConverters.Add(value => value is DateTimeOffset dateTime
+            ? dateTime.ToDutchDateTime().ToString("dd-MM-yyyy HH:mm:ss", _options.CultureInfo)
```

Comment:
Call ' .ToDutchDateTime()' here in the library feels weird. Maybe change it like this:
- add TimeZone to the options with default value of DateTimeOffsetExtensions.WestEuropeZone
- move DateTimeOffsetExtensions to the Core project, add method ConvertToTimeZone and use it here to convert dateTime to options.TimeZone
- use options.TimeZone also below when configuring template context

Add tests testing for example if DateTime with offset 00:00 is printed in configured (WestEurope) time zone.